src/main/java/tds/exam/results/messaging/ExamCompletedMessageListener.java

```diff
@@ -22,41 +22,28 @@
 
     private final ExamService examService;
 
-    private final ExamResultsTransmitterServiceProperties properties;
-
     @Autowired
     public ExamCompletedMessageListener(final ExamResultsService examResultsService,
-                                        final ExamService examService,
-                                        final ExamResultsTransmitterServiceProperties properties) {
```

Comment:
No longer need to pass in `ExamResultsTransmitterServiceProperties` here; retry/recovery is managed by `ExamResultsTransmitterMessagingConfiguration`.